src/osd/scheduler/mClockScheduler.cc

```diff
@@ -119,15 +119,15 @@ WorkItem mClockScheduler::dequeue()
   if (!immediate.empty()) {
     auto ret = std::move(immediate.back());
     immediate.pop_back();
-    return std::move(ret);
+    return ret;
```

Comment:
I recall having made this change after I observed opensuse and possibly one of the ubuntu builds (can't recall which) failing on this line after changing the return type to std::variant for the dequeue() function. Those specific builds probably need to be monitored. I can't provide details of the failures as shaman doesn't show older builds. Is there a way to retrieve details of older shaman builds?

Reply:
@sseshasa not sure how old ? jenkins does keep finite number / days of build logs though. if it's a reproducible build failure, why not reproduce it?

Reply:
@tchaikov Yes, the shaman build with these changes should reproduce it.